### PR TITLE
KN-51614 Translation tool

### PR DESCRIPTION
**Pull request by FromPerm**

### CODE REVIEW

File: EFProvider/LocalizationProvider.cs [hunk 1]
```diff
@@ -14,18 +14,35 @@ namespace Knoema.Localization
...
+				name += scope;
+
+			var result = LocalizationCache.Get<IEnumerable<ILocalizedObject>>(name);
 
 			if (result == null || !result.Any())
 			{
 				using (var context = new LocalizationContext())
 				{
-					result = context.Objects.Where(obj => obj.LocaleId == culture.LCID && obj.Scope.StartsWith("~/")).ToList();
+					if (culture == null)
```
Comment (AlexPhilippov) [inline EFProvider/LocalizationProvider.cs]: I suppose you can reduce ifs here as well

Comment (FromPerm) [reply EFProvider/LocalizationProvider.cs]: Fixed

File: EFProvider/LocalizationProvider.cs [hunk 2]
```diff
@@ -14,18 +14,30 @@ namespace Knoema.Localization
 {
 	public class LocalizationProvider : ILocalizationProvider
 	{
-		public IEnumerable<ILocalizedObject> GetAll(CultureInfo culture)
+		public IEnumerable<ILocalizedObject> GetAll(CultureInfo culture, string scope = null)
 		{
-			var result = LocalizationCache.Get<IEnumerable<ILocalizedObject>>(culture.Name);
+			var name = (culture == null) ? "All" : culture.Name;
```
Comment (AlexPhilippov) [inline EFProvider/LocalizationProvider.cs]: shouldn't go to cache here as well

Comment (AlexPhilippov) [inline EFProvider/LocalizationProvider.cs]: Push All to some constant

Comment (FromPerm) [reply EFProvider/LocalizationProvider.cs]: Not needed

Comment (FromPerm) [reply EFProvider/LocalizationProvider.cs]: Fixed

**Pull request merged**